src/firecracker/src/main.rs

```diff
@@ -93,6 +93,11 @@ fn main() {
                 .takes_value(true)
                 .help("Process start CPU time (wall clock, microseconds). This parameter is optional."),
         )
+        .arg(
+            Argument::new("parent-cpu-time-us")
+                .takes_value(true)
+                .help("Parent process CPU time (wall clock, microseconds). This parameter is optional."),
+        )
```

Comment:
This adds complexity. There's also the option to allow `start_time_cpu_us` to take negative values. It would be simpler, but maybe a bit strage. I'm not sure which option is the cleanest.

Reply:
I did take into consideration allowing negative values for `start_time_cpu_us`, which, indeed, would be much simpler. However, I found it to be too hacky, given the fact that the process CPU time in essence should never take negative values. In my opinion, adding a new parameter made for a cleaner approach.

Reply:
We should just prioritize https://github.com/firecracker-microvm/firecracker/issues/2516 and get rid of all these params.

Reply:
Ok, sounds good to me